### DISCUSSION TITLE
作者请您务必看到此条信息！

### DISCUSSION THREAD

**Discussion by LJ0530** [SailyTeam/Saily#110] [Q&A]
很多人用您的越狱商店会遇到这样的问题，请问如何解决呢？

这项操作无法完成，请参考下面的原因。

adding apt.cydiakk.crackerxi to context 0    
resolving depend firmware (>= 9.0)    
requirement already install firmware    
resolving depend ldid (>=2.0)    
looking for package ldid returns 0    
failed to find ldid (>=2.0), search virtual refrences    
missing ldid (>=2.0)    
🔴 缺失或冲突 ldid (>=2.0)****

**Comment by Lakr233** [comment]
你找个什么软件源 把 ldid 给装上吧
